tests/test_network.py

```diff
@@ -71,6 +71,44 @@ def test_dump_ampl():
             assert file_name in file_names_expected
 
 
+def test_dump_import_iidm():
+    n = pp.network.create_eurostag_tutorial_example1_network()
+    with tempfile.TemporaryDirectory() as tmp_dir_name:
+        tmp_dir_path = pathlib.Path(tmp_dir_name)
+        iidm_file = tmp_dir_path.joinpath('test.xiidm')
+        n.dump(iidm_file, format='XIIDM')
+        file_names = os.listdir(tmp_dir_path)
+        assert len(file_names) == 1
+        assert 'test.xiidm' in file_names
+        n2 = pp.network.load(iidm_file)
+        assert isinstance(n2, pp.network.Network)
+
+
+def test_dump_matpower():
+    n = pp.network.create_eurostag_tutorial_example1_network()
+    with tempfile.TemporaryDirectory() as tmp_dir_name:
+        tmp_dir_path = pathlib.Path(tmp_dir_name)
+        mat_file = tmp_dir_path.joinpath('test.mat')
+        n.dump(mat_file, format='MATPOWER')
+        file_names = os.listdir(tmp_dir_path)
+        assert len(file_names) == 1
+        assert 'test.mat' in file_names
+        n2 = pp.network.load(mat_file)
+        assert isinstance(n2, pp.network.Network)
+
+def test_dump_ucte():
+    n = pp.network.load(str(TEST_DIR.joinpath('test.uct')))
+    with tempfile.TemporaryDirectory() as tmp_dir_name:
+        tmp_dir_path = pathlib.Path(tmp_dir_name)
+        ucte_file = tmp_dir_path.joinpath('test.uct')
+        n.dump(ucte_file, format='UCTE')
+        file_names = os.listdir(tmp_dir_path)
+        assert len(file_names) == 1
+        assert 'test.uct' in file_names
+        n2 = pp.network.load(ucte_file)
+        assert isinstance(n2, pp.network.Network)
```

Comment:
you don't need to test the load again, as you already loaded the file in UCTE format a few lines above